packages/components/search/README.md

```diff
@@ -0,0 +1,156 @@
+<h1 align="center">
+    <a href="https://tradeshift.com/">
+      <img alt="Tradeshift" src="https://tradeshift.com/wp-content/themes/Tradeshift/img/brand/logo-black.png"/>
+    </a>
+</h1>
+
+<h1 align="center">Elements - search</h1>
+
+<p align="center">
+  Part of the reusable Tradeshift UI Components as Web Components.
+    <a href="https://tradeshift.github.io/elements/?path=/story/ts-search--default">
+      Demo
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@tradeshift/elements.search">
+      <img alt="NPM Version" src="https://badgen.net/npm/v/@tradeshift/elements.search" height="20"/>
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.search?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.search" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.search">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.search" height="20"/>
+		</a>
+</p>
+
+<style>
+table {
+    width:100%;
+}
+</style>
+
+## ➤ Properties
+
+| Property    | Attribute   | Type    | Default     | Description                                       |
+| ----------- | ----------- | ------- | ----------- | ------------------------------------------------- |
+| autofocus   | autofocus   | boolean | false       | Shoud the search be auto focused once page loaded |
+| dir         | dir         | string  | ''          | Direction 'rtl' or 'ltr'                          |
```

Comment:
This is weird issue about the `dir` attribute, in out components! If you don't provide the `rtl` it's an `ltr`, but we don't have `ltr` as default :D (I used `ltr` as default value in `tabs` which I'm not sure if it's okay or not as well) What do you guys think is a better idea for a situation like this 🤔 ? @zdlm @aleks-elkin